packages/client/src/app/pages/my-stuff/blog-form/blog-form.component.html

```diff
@@ -46,11 +46,23 @@ <h3>{{ formTitle }}</h3>
             <button class="form-button" (click)="submitForm()">Save</button>
         </div>
         <form [formGroup]="blogForm" (ngSubmit)="submitForm()">
-            <mat-form-field style="width: 100%;" appearance="outline">
-                <mat-label>Title</mat-label>
-                <input matInput placeholder="There's Something I Gotta Say..." formControlName="title">
-            </mat-form-field>
-    
+            <div class="row">
+                <div class="six columns">
+                    <mat-form-field style="width: 100%;" appearance="outline">
+                        <mat-label>Title</mat-label>
+                        <input matInput placeholder="There's Something I Gotta Say..." formControlName="title">
+                    </mat-form-field>
+                </div>
+                <div class="six columns">
+                    <mat-form-field style="width: 100%;" appearance="outline">
+                        <mat-label>Rating</mat-label>
+                        <mat-select [formControlName]="'rating'">
+                            <mat-option *ngFor="let rating of ratings | keyvalue" [value]="rating.key">{{ rating.value }}</mat-option>
```

Comment:
a thought: we really need to sort this in E -> T -> M -> X order at some point. I keep forgetting to do it, but it bugs me every time >.>